app/src/main/java/org/cru/godtools/ui/tooldetails/ToolDetailsFragment.kt

```diff
@@ -131,16 +149,21 @@ class ToolDetailsFragment() :
         val code = tool?.code ?: return
         val locale = translation?.languageCode ?: return
 
-        activity?.run {
-            openToolActivity(code, tool.type, locale, showTips = true)
-            if (settings.getFeatureDiscoveredCount("$FEATURE_TUTORIAL_TIPS$code") < 3) {
-                settings.setFeatureDiscovered("$FEATURE_TUTORIAL_TIPS$code")
-                startTutorialActivity(
-                    PageSet.TIPS,
-                    Bundle().apply { putString(FORMAT_ARG_TOOL_NAME, translation.getName(tool).toString()) }
-                )
-            }
-        }
+        dataModel.tipsLanguage = locale
+        dataModel.tipsTool = code
+        dataModel.tipsType = tool.type
```

Comment:
move these into `launchTrainingTips`, set these values when you are launching the tutorial